tests-integration/src/cluster.rs

```diff
@@ -394,33 +400,45 @@ impl GreptimeDbClusterBuilder {
             Arc::new(InvalidateCacheHandler::new(cache_registry.clone())),
         ]);
 
-        let options = FrontendOptions::default();
+        let mut fe_opts = FrontendOptions::default();
+        // Setup the server addr for local test.
+        fe_opts.grpc.server_addr = format!("127.0.0.1:{}", DEFAULT_GRPC_ADDR_PORT);
```

Comment:
Why configure this? It's the default value. https://github.com/GreptimeTeam/greptimedb/blob/29739b556e6daec6776999b0ef8e93a7b026d3d8/src/servers/src/grpc.rs#L124

Reply:
If we don't configure the `server_addr`, it will use the local IP of the default network device, which does not work in the integration test. We have to specify localhost.